BACKGROUND.md

```diff
@@ -0,0 +1,248 @@
+* `$ref` is "not a network locator, only an identifier"
+* `$ref` limited to schemas
+* Implementations SHOULD NOT assume a network operation for `$ref`
+* Defines `id` as setting the URI and base URI of the schema, as well as plain name fragments
+* Defines initial base URI in terms of RFC 3986
+* Shows JSON Pointer URI fragments that cross `id` as valid schema URIs
+* "Internal references" (SHOULD support) replaces "inline dereferencing"
+* "External references" (SHOULD support) replaces "canonical dereferencing"
+
+### 2017-04-15 JSON Schema Core [draft-wright-json-schema-01](https://datatracker.ietf.org/doc/html/draft-wright-json-schema-01) (draft-06)
+
+* `id` renamed to `$id`
+
+### 2017-07-26 OpenAPI Specification [v3.0.0](https://spec.openapis.org/oas/v3.0.0.html)
+
+* Uses extended subset of JSON Schema draft-wright-json-schema-00, without `id`, `definitions`, or JSON Schema's `$ref`
+* Uses JSON Reference draft-03, both inside and outside of schemas
+* Uses `components` at root level with typed sub-objects instead of `definitions`
+* Base URI not mentioned
+
+### 2017-09-20 AsyncAPI Specification [v1.0.0](https://github.com/asyncapi/spec/blob/1.0.0/README.md)
+
+* Uses extended subset of JSON Schema draft-wright-json-schema-00, without `id`, `definitions`, or JSON Schema's `$ref`
+* Uses JSON Reference draft-03, both inside and outside of schemas
+* Uses `components` at root level with typed sub-objects instead of `definitions`
+* Base URI not mentioned
+
+### 2017-11-19 JSON Schema Core [draft-handrews-json-schema-00](https://datatracker.ietf.org/doc/html/draft-handrews-json-schema-00) (draft-07)
+
+* Wording improvements for `$id`
+
+### 2018-03-19 JSON Schema Core [draft-handrews-json-schema-01](https://datatracker.ietf.org/doc/html/draft-handrews-json-schema-01) (draft-07 clarification)
+
+* `$id`-created plain name fragments described as "location-independent identifiers"
+*  Recommends that the root schema contain an absolute-URI `$id`
+*  Exhaustive schema identification examples showing all possible `$id`-crossing JSON Pointer fragments
+*  Replaced "external referencing" with how and when an implementation might know of a schema from another doucment
+*  Replaced "internal referencing" with how an implementation should recognized schema identifiers during parsing
+*  Dereferencing the former "internal" or "external" references is always the same process
+
+### 2019-08-13 [JSON Reference (draft revival proposal)](https://github.com/hyperjump-io/browser/blob/master/lib/json-reference/README.md)
+
+* Proposes reviving and extending JSON Reference
+* Uses `$href` and `$embedded` for `$ref` and `$id` to avoid collision with JSON Schema
+* Defines behavior in terms of HTTP, web linking, and URLs
+* `$href` based on JSON Reference's `$ref`
+* `$embedded` uses the embedded resource behavior of `$id` in the about-to-be-published draft of JSON Schema
+* `$embedded` restricted to absolute-URIs (with scheme, without fragment)
```

Comment:
Sounds like it's perhaps best if I just take this one out?  Otherwise I'll either be out of alignment with what it says or how its implemented.  Let me know if there's something you want this to say, otherwise I think the history holds up fine without it.